kubernetes/structures_pod.go

```diff
@@ -15,6 +15,8 @@ func flattenPodSpec(in v1.PodSpec) ([]interface{}, error) {
 		att["active_deadline_seconds"] = *in.ActiveDeadlineSeconds
 	}
 
+	att["automount_service_account_token"] = *in.AutomountServiceAccountToken
```

Comment:
`in.AutomountServiceAccountToken` needs a `nil` check before being used, otherwise it can cause the provider to crash with a panic.

Reply:
fixed.